src/main/java/lotto/game/LottoService.java

```diff
@@ -0,0 +1,31 @@
+package lotto.game;
+
+import lotto.ticket.Ticket;
+import lotto.ticket.Tickets;
+import lotto.ticket.strategy.ManualNumbersGenerator;
+import lotto.ticket.strategy.RandomNumbersGenerator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoService {
+
+    public static Tickets buyManualTickets(List<String> manualLottoNumber) {
+        List<Ticket> manualTickets = manualLottoNumber.stream()
+                .map(ManualNumbersGenerator::new)
+                .map(ManualNumbersGenerator::generate)
+                .map(Ticket::new)
+                .collect(Collectors.toList());
+        return new Tickets(manualTickets);
+    }
+
+    public static Tickets buyAutoTickets(LottoCount count) {
+        List<Ticket> tickets = new ArrayList<>();
+        while (count.isGreaterThanZero()) {
+            count = count.decreaseOne();
+            tickets.add(new Ticket(new RandomNumbersGenerator().generate()));
+        }
+        return new Tickets(tickets);
+    }
+}
```

Comment:
```suggestion
public class LottoService {
    private final NumbersGenerator generator;
    public static Tickets buyManualTickets(List<String> manualLottoNumber) {
        List<Ticket> manualTickets = manualLottoNumber.stream()
                .map(ManualNumbersGenerator::new)
                .map(ManualNumbersGenerator::generate)
                .map(Ticket::new)
                .collect(Collectors.toList());
        return new Tickets(manualTickets);
    }
    public static Tickets buyAutoTickets(LottoCount count) {
        List<Ticket> tickets = new ArrayList<>();
        while (count.isGreaterThanZero()) {
            count = count.decreaseOne();
            tickets.add(new Ticket(generator.generate()));
        }
        return new Tickets(tickets);
    }
}
```

인터페이스의 장점이 무엇인지 고민해보고 어떠한 차이가 있을지 고민해보아요!